package/MDAnalysis/analysis/base.py

```diff
@@ -198,25 +248,33 @@ def run(self, start=None, stop=None, step=None, verbose=None):
 
 
 class AnalysisFromFunction(AnalysisBase):
-    """
-    Create an analysis from a function working on AtomGroups
+    r"""Create an :class:`AnalysisBase` from a function working on AtomGroups
 
-    Attributes
-    ----------
-    results : ndarray
-        results of calculation are stored after call to ``run``
+    .. code-block:: python
```

Comment:
This should still be under an Example heading.

And Examples come after Attributes etc, according to https://userguide.mdanalysis.org/stable/contributing_code.html#guidelines-for-writing-docstrings and the numpydoc standard https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard